kong.yml

```diff
@@ -90,7 +91,7 @@ nginx: |
     real_ip_recursive on;
 
     # Other Settings
-    client_max_body_size 128m;
+    client_max_body_size 0m;
```

Comment:
does NGINX has any default value for this field or its upto user to set this value?